keps/sig-cloud-provider/providers/0005-cloud-provider-alibaba-cloud.md

```diff
@@ -0,0 +1,113 @@
+---
+kep-number: 5
+title: Cloud Provider for Alibaba Cloud
+authors:
+  - "@aoxn"
+owning-sig: sig-cloud-provider
+participating-sigs:
+  - nil
+reviewers:
+  - TBD
+  - "@alicedoe"
+approvers:
+  - "@andrewsykim"
+  - "@hogepodge"
+  - "@jagosan"
+editor: TBD
+creation-date: 2018-06-20
+last-updated: 2018-06-20
+status: provisional
+see-also:
+  - KEP-2
+  - KEP-4
+replaces:
+  - KEP-3
+superseded-by:
+  - KEP-100
+---
+
+# Cloud Provider for Alibaba Cloud
+
+This is a KEP for adding ```Cloud Provider for Alibaba Cloud``` into the Kubernetes ecosystem.
+
+## Table of Contents
+
+* [Table of Contents](#table-of-contents)
+* [Summary](#summary)
+* [Motivation](#motivation)
+    * [Goals](#goals)
+    * [Non-Goals](#non-goals)
+* [Requirements](#requirements)
+* [Proposal](#proposal)
+
+## Summary
+
+Alibaba Cloud provides the Cloud Provider interface implementation as an out-of-tree cloud-controller-manager. It allows Kubernetes clusters to leverage the infrastructure services of Alibaba Cloud .
+It is original open sourced project is [https://github.com/AliyunContainerService/alicloud-controller-manager](https://github.com/AliyunContainerService/alicloud-controller-manager)
+
+## Motivation
+
+### Goals
+
+Cloud Provider of Alibaba Cloud  implements interoperability between Kubernetes cluster and Alibaba Cloud. In this project, we will dedicated in:
+- Provide reliable, secure and optimized integration with Alibaba Cloud for Kubernetes
+
+- Help on the improvement for decoupling cloud provider specifics from Kubernetes implementation.
+
+   
+
+### Non-Goals
+
+The networking and storage support of Alibaba Cloud for Kubernetes will be provided by other projects. 
+
+E.g. 
+
+* [Flannel network for Alibaba Cloud VPC](https://github.com/coreos/flannel)
+* [FlexVolume for Alibaba Cloud](https://github.com/AliyunContainerService/flexvolume)
+
+
+## Prerequisites
+
+1. The VPC network is supported in this project. The support for classic network or none ECS environment will be out-of-scope. 
+2. When using the instance profile for authentication, an instance role is required to attach to the ECS instance firstly.
+3. Kubernetes version v1.7 or higher
+
+### Repository Requirements
+
+Alibaba Cloud Providers is a working implementation of the [Kubernetes Cloud Controller Manager](https://kubernetes.io/docs/tasks/administer-cluster/running-cloud-controller/).
+
+The repo requirements is mainly a copy from [cloudprovider KEP](https://github.com/kubernetes/community/blob/master/keps/sig-cloud-provider/0002-cloud-controller-manager.md#repository-requirements). Open the link for more detail.
+
+### User Experience Reports
+
+Usage of aliyun container services can be seen from github issues in the existing alicloud controller manager repo: https://github.com/AliyunContainerService/alicloud-controller-manager/issues
```

Comment:
It may not hurt to mention Alibaba's CNCF membership, that's usually also a pretty good indication of user adoption and support in the community.